lib/basedriver/commands/session.js

```diff
@@ -2,16 +2,26 @@ import _ from 'lodash';
 import log from '../logger';
 import { errors } from '../../mjsonwp';
 import UUID from 'uuid-js';
+import { processCapabilities } from '../capabilities';
 
 let commands = {};
 
-commands.createSession = async function (caps) {
+commands.createSession = async function (desiredCapabilities, requiredCaps, capabilities) {
```

Comment:
as long as a client could upgrade to a version of base driver containing this change and not have to change anything in order to keep functioning the way it was, then we're good